nntrainer/utils/ini_wrapper.cpp

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file ini_wrapper.cpp
+ * @date 08 April 2021
+ * @brief NNTrainer Ini Wrapper helps to save ini
```

Comment:
is this going to be used to serialize the `GraphRepresentation` to ini format?

Reply:
Yes.

Reply:
Please add to description.

Reply:
done